### PR TITLE
Fix/Log blobstor delete operation

### PR DESCRIPTION
**Pull request by carpawell**
Log message fixes related to the `blobstor.Delete` operation.

### CODE REVIEW

File: pkg/local_object_storage/internal/log/log.go [hunk 1]
```diff
@@ -24,3 +24,13 @@ func AddressField(addr interface{}) zap.Field {
 func OpField(op string) zap.Field {
 	return zap.String("op", op)
 }
+
+// TypeField returns logger's field for storage type.
+func TypeField(typ string) zap.Field {
```
Comment (cthulhu-rider) [inline pkg/local_object_storage/internal/log/log.go:29]:
```suggestion
func StorageTypeField(typ string) zap.Field {
```

**Review by cthulhu-rider — APPROVED** [review]

File: pkg/local_object_storage/internal/log/log.go [hunk 2]
```diff
@@ -24,3 +24,13 @@ func AddressField(addr interface{}) zap.Field {
...
 }
+
+// StorageTypeField returns logger's field for storage type.
+func StorageTypeField(typ string) zap.Field {
+	return zap.String("type", typ)
+}
+
+// StorageIDField returns logger's field for storage ID.
+func StorageIDField(id []byte) zap.Field {
+	return zap.String("storageID", string(id))
```
Comment (fyrchik) [inline pkg/local_object_storage/internal/log/log.go:35]: Let's stay on `storage_id` in logs? Like we already do for `shard_id`

File: pkg/local_object_storage/blobstor/delete.go [hunk 3]
```diff
@@ -15,18 +13,25 @@ func (b *BlobStor) Delete(prm common.DeletePrm) (common.DeleteRes, error) {
 			res, err := b.storage[i].Storage.Delete(prm)
 			if err == nil || !errors.As(err, new(apistatus.ObjectNotFound)) {
 				if err == nil {
-					storagelog.Write(b.log,
-						storagelog.AddressField(prm.Address),
-						storagelog.OpField("DELETE"),
-						zap.String("type", b.storage[i].Storage.Type()),
-						zap.String("storage ID", string(prm.StorageID)))
+					logOp(b.log, deleteOp, prm.Address, b.storage[i].Storage.Type(), nil)
```
Comment (fyrchik) [inline pkg/local_object_storage/blobstor/delete.go:16]: It was `prm.StorageID` before.

Comment (carpawell) [reply pkg/local_object_storage/blobstor/delete.go]: yes, but it is in the `if prm.StorageID == nil` branch. ok, retuned `prm.StorageID`

File: pkg/local_object_storage/internal/log/log.go (hunk 2 above)
Comment (carpawell) [reply pkg/local_object_storage/internal/log/log.go]: agree, changed

**Review by fyrchik — APPROVED** [review]

**Pull request merged**